tokumx/datadog_checks/tokumx/vendor/vendor.txt

```diff
@@ -0,0 +1 @@
+pymongo==3.5.1
```

Comment:
Need this?

Reply:
I included this as it was part of the requirements for the vendor script I used. I don't have a strong opinion either way, but I like having the reference for what version of pymongo is being vendored here.